packages/@o3r/configuration/src/tools/configuration.signal.ts

```diff
@@ -0,0 +1,37 @@
+import type { InputSignal, Signal } from '@angular/core';
+import { toObservable, toSignal } from '@angular/core/rxjs-interop';
+import type { Configuration } from '@o3r/core';
+import { shareReplay } from 'rxjs';
+import { getConfiguration } from '../core';
+import type { ConfigurationBaseService } from '../services';
+
+/** Configuration signal */
+export type ConfigurationSignal<T> = Signal<T> & { configId: string };
+
+/**
+ * Get a configuration signal
+ * @param configInput
+ * @param configId
+ * @param defaultConfig
+ * @param configurationService
+ */
+export const configSignal = <T extends Configuration>(
```

Comment:
Would make maybe more sense to be called `createConfigSignal` (or `generateConfigSignal` or `getConfigSignal`) to follow standard function naming and make sure that it can not be misleading

Reply:
I followed the Angular naming convention like `input()`, `signal()`, etc...